vdirsyncer/storage/google.py

```diff
@@ -94,6 +94,11 @@ def _save_token(token):
 class GoogleCalendarStorage(dav.CaldavStorage):
     __doc__ = '''Google calendar.
 
+    .. note:: You have to poperly set up ``collections``, otherwise errors like
```

Comment:
At this point I'm actually curious which problem led you to write this. Stuff like #463? Though that typo should be properly caught when parsing the config...